front-end/src/utils/apiCalls.js

```diff
@@ -13,6 +13,16 @@ export const postNewRecipe = (recipe, userId) => {
     });
 };
 
+export const searchForFriends = (userSearchTerm) => {
+  return axios.get(`http://localhost:3000/friends/${userSearchTerm}`)
+    .then((res) => {
+      return res;
+    })
+    .catch((err) =>{
+      return err;
```

Comment:
Was going to say that returning the actual error could be problematic where you are calling this function, but you handled it beautifully with displaying data from state if status code is not 200!

Reply:
Thanks Rick, wasn’t sure about my error handling as well and am always open for suggestion